src-backbone/app/js/views/procedures/account_navbar_view.js

```diff
@@ -0,0 +1,21 @@
+module.exports = Marionette.ItemView.extend({
+
+    template: Handlebars.templates.account_navbar,
+
+    constructor: function(options) {
+        this.app = (options && options.app) || global.App;
+        Marionette.ItemView.prototype.constructor.call(this, options);
+    },
+
+    templateHelpers: function() {
+        var self = this;
+
+        return {
+            username: function() {
+                // TODO fetch from server
+                return 'Trinovantes';
```

Comment:
Yea we need an `/api/account` endpoint for getting info and changing email/passwords